config/routes.rb

```diff
@@ -30,6 +30,23 @@
 
   resources :custom_manifests, only: [:create, :update, :destroy, :show]
 
+  resources :tool_groups, path: "tool-groups", only: [:create, :destroy, :index, :show, :update]
+
+  # Rule Languages
+  post "tool-groups/:id/rule-languages", to: "rule_languages#create"
```

Comment:
the `:id` parameter here doesn't match the `:tool_group_id` parameter you are using within `rule_languages#create`, this yields a "tool_group must exist" error

Reply:
I think it is solved when sent code in commit [0ee2a267b7b03c7a9480b4f865809dddd33b5413](https://github.com/CruGlobal/mobile-content-api/pull/1326/commits/0ee2a267b7b03c7a9480b4f865809dddd33b5413#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5R36)